vcluster/configure/vcluster-yaml/sync/to-host/advanced/custom-resources.mdx

```diff
@@ -28,10 +28,6 @@ vCluster also adds the necessary cluster and namespace-level RBAC permissions to
 This feature currently only works for namespace-scoped resources only.
 :::
 
-:::info Multi-Namespace-Mode
```

Comment:
Why was this removed? The title of the info box is wrong, but it's pointing to the correct/new docs.